get_asteroidcount.py

```diff
@@ -0,0 +1,41 @@
+def get_NEA_count(by_method: bool = True,
+                        by_class: bool = True) -> dict:
+    """
+    Get the current total number of confirmed exoplanets from the NASA
+    Exoplanet Archive. Additionally, also get the number of exoplanets
+    grouped by detection method or by planet class (if desired).
+    Args:
+        by_method: Whether or not to also return the exoplanet count
+            grouped by detection method.
+        by_class: Whether or not to also return the exoplanet count
+            grouped by (approximate) planet class.
+    Returns:
+        A dictionary with keys `{"total", "by_method", "by_class"}`
+        which holds the respective count of exoplanets.
+    """
+
+    # Get all potentially hazardous asteroids from MPC
+    PHA = MPC.query_objects('asteroid', pha=1)
```

Comment:
We should add a try/except in case there's no internet connection.

Reply:
We could maybe even try to introduce some sort of system which marks widgets as "offline ready" or not?

Reply:
Oo great idea. Raise it as an issue, and then we can remember to come back to it later.

Reply:
This is something we should discuss as a policy for the group, but a message like that looks good.  We could also return that as a string representation so that the user sees this as part of the output when using the program.